### PR TITLE
Added Try what you'll build section

### PR DESCRIPTION
**Pull request by justineechen**

### CODE REVIEW

**Comment by justineechen** [comment]
Addressing OpenLiberty/guides-common#244

File: README.adoc [hunk 1]
```diff
@@ -143,6 +143,33 @@ convention has the link as the value of the relationship:
 [role=command]
 include::{common-includes}/gitclone.adoc[]
 
+=== Try what you'll build
+
+To try out the application, first navigate to the `finish` directory and then run the following Maven goals to build the application and run it inside Open Liberty:
```
Comment (evelinec) [inline README.adoc]: Use common adoc, where applicable.

File: README.adoc [hunk 2]
```diff
@@ -143,6 +143,33 @@ convention has the link as the value of the relationship:
...
+[role='command']
+```
+mvn install
+```
+
+This command builds the application and creates a `.war` file in the target directory. It also
```
Comment (evelinec) [inline README.adoc]: This paragraph is not needed.

File: README.adoc [hunk 3]
```diff
@@ -143,6 +143,33 @@ convention has the link as the value of the relationship:
...
+configures and installs Open Liberty into the `target/liberty/wlp` directory.
+
+Next, run the Maven `liberty:start-server` goal:
+[role='command']
+```
+mvn liberty:start-server
+```
+
+This goal starts an Open Liberty server instance. Your Maven `pom.xml` is already configured to start
```
Comment (evelinec) [inline README.adoc]: This paragraph is not needed.

File: README.adoc [hunk 4]
```diff
@@ -143,6 +143,33 @@ convention has the link as the value of the relationship:
...
+the application in this server instance.
+
+After the server runs, you can find your hypermedia-driven `inventory` service at the following URL:
+
+* http://localhost:9080/inventory/hosts[http://localhost:9080/inventory/hosts^]
```
Comment (evelinec) [inline README.adoc]: I think this syntax works `http://localhost:9080/inventory/hosts[^]`.

**Pull request merged**